src/Examples/SequenceToSequence.cs

```diff
@@ -137,7 +137,7 @@ private static void train(int epoch, Tensor train_data, TransformerModel model,
                         total_loss += loss.to(torch.CPU).item<float>();
                     }
 
-                    if (batch % log_interval == 0 && batch > 0) {
+                    if (batch % log_interval == 0 && batch > 0 || (batch == tdlen / bptt)) {
```

Comment:
nit: would be good if you add parentheses to these conditions to show the intention how the conditions  should be evaluated.